packages/components/src/highlight-cards/annual-highlight-cards.tsx

```diff
@@ -42,33 +44,41 @@ export default function AnnualHighlightCards( {
 
 	return (
 		<div className={ classNames( 'highlight-cards', className ?? null ) }>
-			{ header }
+			<div className="highlight-year-navigation">
+				{ header }
+				{ navigation && navigation }
+			</div>
 
 			<div className="highlight-cards-list">
 				<HighlightCard
 					heading={ translate( 'Posts' ) }
 					icon={ <Icon icon={ postContent } /> }
 					count={ counts?.posts ?? null }
+					showValueTooltip
```

Comment:
And a few others below too

Reply:
That's right. I was hoping to merge #71159 first so once I update this PR it will be already on `trunk` and won't show up as changes here. Sorry for not mentioning it earlier.